subtests/docker_cli/kill/kill_utils.py

```diff
@@ -129,7 +129,8 @@ def initialize(self):
         else:
             self._init_container_normal(name)
 
-        time.sleep(self.config['wait_start'])
+        cmd = self.sub_stuff['container_cmd']
+        cmd.wait_for_ready(timeout=self.config['wait_start'])
```

Comment:
There's another copy of ``kill_utils.py`` that maybe needs this update as well.  I can't remember why the subtests were split in two - requiring duplicating the utils code.  I have an issue open to merge them, but maybe there's a better way to fix the problem.  Until that appears, it's probably best to at least keep the two files in-sync. with each-other.